nefertari/elasticsearch.py

```diff
@@ -1,13 +1,16 @@
 from __future__ import absolute_import
 import json
+import threading
+from collections import OrderedDict
 import logging
 from functools import partial
 from collections import defaultdict
 import os
+from datetime import datetime
 
 
 import elasticsearch
-from elasticsearch.exceptions import ElasticsearchException, TransportError
+from elasticsearch.exceptions import TransportError, ElasticsearchException
```

Comment:
During development I've added some exception for investigation and after that I've removed it.